src/cdk/overlay/dispatchers/overlay-outside-click-dispatcher.spec.ts

```diff
@@ -125,26 +125,34 @@ describe('OverlayOutsideClickDispatcher', () => {
     overlayTwo.dispose();
   });
 
-  it(`should not dispatch click event when click on element
-      included in excludeFromOutsideClick array`, () => {
+  it('should dispatch the click event when click is on an element outside the overlay', () => {
+    const portal = new ComponentPortal(TestComponent);
     const overlayRef = overlay.create();
-    const spy = jasmine.createSpy('overlay mouse click event spy');
+    overlayRef.attach(portal);
+    const button = document.createElement('button');
+    document.body.appendChild(button);
+
+    const spy = jasmine.createSpy('overlay mouse click spy');
     overlayRef.outsidePointerEvents().subscribe(spy);
 
-    const overlayConfig = overlayRef.getConfig();
-    expect(overlayConfig.excludeFromOutsideClick).toBeDefined();
-    expect(overlayConfig.excludeFromOutsideClick!.length).toBe(0);
+    button.click();
+    expect(spy).toHaveBeenCalled();
 
-    overlayRef.attach(new ComponentPortal(TestComponent));
+    button.parentNode!.removeChild(button);
+    overlayRef.dispose();
```

Comment:
Should these disposals happen in an afterEach() block, so they happen even if the test fails?

Reply:
Slightly tricky because different tests create a different number of overlays; it would probably be better to de-dupe the dispose calls, but I wouldn't hold up the PR on it since it was already like this

Reply:
@teflonwaffles  I agree that it should be cleaned up, but I opted to keep with the structure of the existing tests rather than polluting this PR with a test refactor.